index.html

```diff
@@ -38,6 +39,156 @@
     </div>
 
   </section>
+
+  <section class="work-section">
+    <div class="RWbox"><div class="RWork"><p>My Recent works</p>
+    <div class="workline"></div></div>
+    </div>
+    <div class="work-container">
```

Comment:
- [ ] Kindly ensure that the main container of the projects uses `display:grid` to layout the project cards which is to ensure that the section projects cards are responsive and also gives you flexibility in positioning them 👍🏽

Reply:
I've used grid in the container for different work cards can you be more specific, please
![Screenshot (6)](https://user-images.githubusercontent.com/103601335/192104670-339cd1cf-58ef-4071-b6c9-4092d155fab3.png)

Reply:
@Cluab Good job, I didn't spot it initially. Sorry for that 🙏🏽 

You can proceed to fix other changes requested 👍🏽